config/e2e/filebeat.yaml

```diff
@@ -231,3 +232,39 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: filebeat
+---
+{{ if .OcpCluster }}
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: filebeat
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: true
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: true
+allowedCapabilities: []
+defaultAddCapabilities: []
+fsGroup:
+  type: RunAsAny
+groups:
+  - system:cluster-admins
```

Comment:
nit: does these 2 groups really need to belong to this SCC ?

Reply:
They do not. Fixed.